events/ready.js

```diff
@@ -49,21 +49,25 @@ module.exports = {
                         !member.user.bot
                 ).size;
                 console.log(onlineUsers);
-                if (!dbguild.onlineChannelId) return;
-                guild.channels.edit(dbguild.onlineChannelId, {
-                    name: `Online: ${onlineUsers}`,
-                });
-                if (!dbguild.allChannelId) return;
-                guild.channels.edit(dbguild.allChannelId, {
-                    name: `Members: ${guild.memberCount}`,
-                });
-                if (!dbguild.botChannelId) return;
-                guild.channels.edit(dbguild.botChannelId, {
-                    name: `Bots: ${
-                        guild.members.cache.filter((member) => member.user.bot)
-                            .size
-                    }`,
-                });
+                if (dbguild.onlineChannelId) {
+                    guild.channels.edit(dbguild.onlineChannelId, {
+                        name: `Online: ${onlineUsers}`,
+                    });
+                }
+                if (dbguild.allChannelId) {
+                    guild.channels.edit(dbguild.allChannelId, {
+                        name: `Members: ${guild.memberCount}`,
+                    });
+                }
+                if (dbguild.botChannelId) {
+                    guild.channels.edit(dbguild.botChannelId, {
+                        name: `Bots: ${
+                            guild.members.cache.filter(
+                                (member) => member.user.bot
+                            ).size
+                        }`,
+                    });
+                }
```

Comment:
i feel like in the future this entire thing could be improved (for readability, mostly) by reducing repetition. we're basically doing the same thing three times here, so that could just be its own function

Reply:
noted